packages/cli/cli/src/persistence/getPathToOutputDirectoriesFile.ts

```diff
@@ -0,0 +1,13 @@
+import { AbsoluteFilePath, join, RelativeFilePath } from "@fern-api/fs-utils";
+import { homedir } from "os";
+
+const APPROVED_OUTPUT_DIRECTORIES_FILENAME = "approved-output-directories";
+const LOCAL_STORAGE_FOLDER = process.env.LOCAL_STORAGE_FOLDER ?? ".fern";
+
+export function getPathToOutputDirectoriesFile(): AbsoluteFilePath {
```

Comment:
nit: `getPathToAllowedDirectoriesFile`

Reply:
I prefer output directories to allowed directories - particularly if we autosave all output directories so that they are allowed (based on later comment to remove the 2nd prompt)